hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1359,6 +1363,54 @@ public static com.hederahashgraph.api.proto.java.FileID fromPbj(FileID someFileI
                 .build();
     }
 
+    public static com.hederahashgraph.api.proto.java.CustomFee fromPbj(CustomFee customFee) {
+        var builder = com.hederahashgraph.api.proto.java.CustomFee.newBuilder();
+        if (customFee.hasFixedFee()) {
+            builder.setFixedFee(fromPbj(customFee.fixedFee()));
+        } else if (customFee.hasFractionalFee()) {
+            builder.setFractionalFee(fromPbj(customFee.fractionalFee()));
+        } else if (customFee.hasRoyaltyFee()) {
+            builder.setRoyaltyFee(fromPbj(customFee.royaltyFee()));
+        }
+
+        builder.setFeeCollectorAccountId(fromPbj(customFee.feeCollectorAccountId()));
+        builder.setAllCollectorsAreExempt(customFee.allCollectorsAreExempt());
+
+        return builder.build();
+    }
+
+    public static RoyaltyFee fromPbj(com.hedera.hapi.node.transaction.RoyaltyFee royaltyFee) {
+        var builder = com.hederahashgraph.api.proto.java.RoyaltyFee.newBuilder();
+        builder.setExchangeValueFraction(fromPbj(royaltyFee.exchangeValueFraction()));
+        if (royaltyFee.hasFallbackFee()) builder.setFallbackFee(fromPbj(royaltyFee.fallbackFee()));
+        return builder.build();
+    }
+
+    public static Fraction fromPbj(com.hedera.hapi.node.base.Fraction fraction) {
+        var builder = com.hederahashgraph.api.proto.java.Fraction.newBuilder();
+        builder.setNumerator(fraction.numerator());
+        builder.setDenominator(fraction.denominator());
+        return builder.build();
+    }
+
+    public static FractionalFee fromPbj(com.hedera.hapi.node.transaction.FractionalFee fractionalFee) {
+        var builder = com.hederahashgraph.api.proto.java.FractionalFee.newBuilder();
+        builder.setFractionalAmount(fromPbj(fractionalFee.fractionalAmount()));
+        builder.setMinimumAmount(fractionalFee.minimumAmount());
+        builder.setMaximumAmount(fractionalFee.maximumAmount());
+        builder.setNetOfTransfers(fractionalFee.netOfTransfers());
+        return builder.build();
+    }
+
+    public static FixedFee fromPbj(com.hedera.hapi.node.transaction.FixedFee fixedFee) {
```

Comment:
this is missing `@NonNull` annotation on return, as well as `@Nullable` on the parameter.

Reply:
done thanks